mindmeld/models/nn_utils/helpers.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+try:
+    import torch
+except ImportError:
+    pass
+
+logger = logging.getLogger(__name__)
+
+
+# utils
+
+def get_disk_space_of_model(pytorch_module):
+    filename = "temp.bin"
```

Comment:
> You can use sys.getsizeof instead of writing to disk to check size of the object.

based on the description here https://docs.python.org/2/library/sys.html#sys.getsizeof and from simple check with a bert-base model, found both solutions gives different results

- using `sys.getsizeof`: 19880
- using `get_disk_space_of_model`: 433325247 (i.e. 433 MB) 

the latter being correct result!

> This will prevent unnecessary spikes in memory to write to disk
The quantity of memory is anyway used when actually saving the pytorch model. So in that sense, it is not using additional disk space anyway. But I'll make sure that this computation of disk space method is called only when logging level is set to info or if the verbose flag is on.